src/components/TradeNav.js

```diff
@@ -14,7 +14,7 @@ const StyledTradeIcon = styled(TradeIcon)`
 const TradeNav = ({ ...props }) => {
   const { lang } = useLanguage();
   return (
-    <Link href={`/${Routes.TRADE}`} target="_blank">
```

Comment:
What's the reason for removing this?

Reply:
Rich requested this change so that it doesn't open a new window when going back and forth between Borrow and Trade. I concede that it doesn't belong to this PR, but was asked to just "throw it in, now that you're at it" sort of thing.

Reply:
Ya that's ok, I'd run it by Chris just in case as he's PO.